### ISSUE TITLE
[CAIA a11y support] Staging Prep Assistance for CAIA - Decision Reviews (Accessibility Testing)

### ISSUE THREAD

**Issue by eli-oat** [department-of-veterans-affairs/va.gov-team#73147]
### Product information

- [X] Team name, product name, and feature name have been added to the title of this issue.
- [X] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

@eli-oat

### Use of color and color contrast checks

- [X] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [X] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [X] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [X] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [X] Any charts, maps, infographics, and tables convey all information without only relying on color
- [X] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### How did color testing go?

NOTE testing was preformed through a managed instance of Chrome on CAG. Was unable to use Colorblindly, used WCAG color contrast checker instead. 

Screenshot of those results included

<img width="371" alt="A screenshot of the WCAG color contrast checker" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/3342274/2bd7305c-faa0-4e0c-bccf-2976dfecef20">

### Who completed the axe scans?

@eli-oat

### axe checks

- [X] Each page has been scanned using axe (results shared below)
- [ ] axe is integrated into your end-to-end testing

### axe DevTools scan results

No issues

### How did axe testing go?

No issues

### Who completed the content zoom and reflow test?

@eli-oat

### Content zoom and reflow checks

- [X] All page elements are readable and usable at 200% zoom
- [X] All page elements are readable and usable at 300% zoom
- [X] All page elements are readable and usable at 400% zoom

### How did content zoom and reflow testing go?

No issues, though I did notice that long link text in the "on this page" section may be sort of overly aggressively wrapped at 300% and 400% zoom. 

### Who completed the keyboard navigation test?

@eli-oat

### Keyboard navigation checks

- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [X] All elements under focus have a visible focus indicator
- [X] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

Many of the links don't function, but I presume this is because I'm testing a preview page?
